bump-version.sh

```diff
@@ -1,43 +0,0 @@
-#!/bin/bash
```

Comment:
If we switch to a python script, also the instructions in this file should be updated accordingly:
https://github.com/ROOT-Sim/core/blob/f6618296aadbeef5eb9230a8dc9fcf7490838d27/docs/CONTRIBUTING.md

Reply:
This will be fixed in the next commit!